problem_builder/public/js/lms_util.js

```diff
@@ -0,0 +1,109 @@
+(function () {
```

Comment:
@jbzdak Is there a reason not to move the contents of this file to the existing `util.js` file? If they do need to be kept in a separate file, I think it would make sense to call it `step_util.js` to indicate the relationship more clearly. (This would also take into account the fact that usage of this file isn't limited to the LMS runtime.)